.github/workflows/ci-repo.yml

```
@@ -173,28 +173,128 @@
+            dbus-x11 \
+            xfonts-100dpi \
+            xfonts-75dpi \
+            libxrender1 \
+            libxext6 \
+            libx11-6 \
+            xfonts-base \
+            nickle cairo-5c \
+            xorg-docs-core
+
+      - name: Set DISPLAY environment variable
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:23: Double quote to prevent globbing and word splitting

(shellcheck)